packages/kit/src/runtime/server/page/render.js

```diff
@@ -317,6 +322,19 @@ export async function render_response({
 			}
 		}
 
+		if (manifest._.client.routes && state.prerendering && !state.prerendering.fallback) {
+			const route_resolution_url = regular_route_to_route_resolution(event.url, options);
```

Comment:
having a `regular_route_to_route_resolution` function that wraps `to_route_resolution` feels like unnecessary indirection

```suggestion
			const route_resolution_url = to_route_resolution(event.url, paths.base, options.app_dir);
```

Reply:
No, because it makes it possible to have all the logic related to this in the backend point to one file. Everything flows through that which I think is very benefitial

Reply:
Not sure I agree here. The functions for detecting/adding/removing the resolution prefix are essentially mirrors of the `[verb]_data_suffix` functions; I think it makes more sense to have the pathname manipulation stuff together in one place (shared between client and server), and all the stuff for actually generating the responses in a different module. I have some changes locally and I think you'll agree with them when you see — will push them up shortly